lib/src/main/java/uk/gov/di/ipv/core/library/service/CiStorageService.java

```diff
@@ -100,25 +132,186 @@ public void submitMitigatingVcList(
     public List<ContraIndicatorItem> getCIs(
             String userId, String govukSigninJourneyId, String ipAddress)
             throws CiRetrievalException {
+        InvokeResult result =
+                invokeClientToGetCIResult(
+                        CI_STORAGE_GET_LAMBDA_ARN,
+                        govukSigninJourneyId,
+                        ipAddress,
+                        userId,
+                        "Retrieving CI's from CI storage system.");
+        String jsonResponse = new String(result.getPayload().array(), StandardCharsets.UTF_8);
+        GetCiResponse response = gson.fromJson(jsonResponse, GetCiResponse.class);
+        return response.getContraIndicators();
+    }
+
+    public ContraIndications getContraIndicatorsVC(
+            String userId, String govukSigninJourneyId, String ipAddress)
+            throws CiRetrievalException {
+        InvokeResult result =
+                invokeClientToGetCIResult(
+                        CIMIT_GET_CONTRAINDICATORS_LAMBDA_ARN,
+                        govukSigninJourneyId,
+                        ipAddress,
+                        userId,
+                        "Retrieving CI's from CIMIT system.");
+        SignedJWT ciSignedJWT = extractAndValidateContraIndicatorsJwt(result, userId);
+        ContraIndicatorEvidenceDto contraIndicatorEvidence =
+                parseContraIndicatorEvidence(ciSignedJWT);
+
+        return mapToContraIndications(contraIndicatorEvidence);
+    }
+
+    private InvokeResult invokeClientToGetCIResult(
+            EnvironmentVariable cimitGetContraindicatorsLambdaArn,
```

Comment:
can use generic parameter name for both arn.

Reply:
Done